packages/eslint-plugin/docs/rules/no-dynamic-i18n-messages.md

```diff
@@ -0,0 +1,42 @@
+# enforce translate calls to be plain text labels (no-dynamic-i18n-messages)
```

Comment:
I'm thinking about what's the best practice to sync this with our website. We may want to break from ESLint norms here and put the docs within `website/docs/api` instead.

Reply:
Yeah, this needs to be addressed in the current PR. Thanks!

Reply:
Addressed in aa58521.